lib/core/file.nit

```diff
@@ -511,8 +511,8 @@ class Path
 		var output = dest.open_wo
 
 		while not input.eof do
-			var buffer = input.read_bytes(1024)
-			output.write_bytes buffer
+			var buf = input.read_bytes(4096)
+			output.write_bytes buf
```

Comment:
why did you rename `buffer` to `buf`?

Reply:
Artefacts of reworking it all, I'll leave the `buffer` name